public/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```

Comment:
It's a shame you didn't have a name for your app!  A favicon would also have been nice.  Did you just run out of time?

Reply:
totally forgot hahaa i've learnt to ignore the "404 favicon missing" message 😂